app/assets/javascripts/govuk_publishing_components/components/cookie-banner.js

```diff
@@ -34,5 +55,19 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
     }
   }
 
+  CookieBanner.prototype.setCookieConsent = function () {
+    window.GOVUK.approveAllCookieTypes()
+    this.$module.showConfirmationMessage()
+    window.GOVUK.setCookie('seen_cookie_message', 'true')
+  }
+
+  CookieBanner.prototype.showConfirmationMessage = function () {
+    this.$cookieBannerMainContent = this.$module.querySelector('div[data-cookie-banner-main]')
```

Comment:
I consider it safe to use the BEM classes for querySelectors, no need to extend with data attributes unless we need a key/value pair to use within the script. Again something that's not yet written in the style guide probably.